qtdialogs/DlgUnlockWallet.py

```diff
@@ -343,3 +359,25 @@ def reply(self, passphrase):
       packet.success = bool(len(passphrase) != 0)
       packet.unlockRequest = passphrase
       super().reply()
+
+################################################################################
+class DlgUnlockMigratingWallet(DlgUnlockWallet):
```

Comment:
This class is a copy of the one in DlgMigrateWallet
It is never used, hence the `_suppressUnlockError` member being added into the parent class.